test/function_test.py

```diff
@@ -872,3 +872,27 @@ def f(x):
         @batched(max_batch_size=1, wait_ms=1)
         def g(x=1):
             return [x_i**2 for x_i in x]
+
+
+@pytest.mark.parametrize("feature_flag_enabled", [True, False, None])
+def test_extended_input_queue_feature_flag(client, servicer, feature_flag_enabled):
+    app = App()
+    dummy_modal = app.function()(dummy)
+
+    if feature_flag_enabled == True:
+        kwargs = {"use_extended_input_queue": True}
+    elif feature_flag_enabled == False:
+        kwargs = {"use_extended_input_queue": False}
+    else:
+        kwargs = {}
```

Comment:
Let me know if there's a better way to implement the feature flag. Using annotations was mentioned as another possible option.

Reply:
Also open to suggestions for a better name for the flag.

Reply:
I like the name. What was the annotations suggestion?

Reply:
I think environment variable might be a slightly more common way to toggle a feature flag? But definitely easier to unit test this way :D

Reply:
Oh I should have said "function decorator" instead of "annotation" - my brain reverted to java :) It was something @gongy mentioned in step 2 [here](https://linear.app/modal-labs/project/durable-execution-fc18c514138d/overview#heading-165bb51c).

I assume it would be added to the modal function - but it seems like this is more a property of the spawn method, than the function being invoked.

Happy to go with environment variable if that is more standard in python.

Reply:
Pushed another commit changing it to use env vars. @gongy pointed out a downside of passing the flag to the spawn method I hadn't considered - although unlikely, the name could collide with a user variable, and in general mixing our variables with theirs seems not good.